microsoft-365/security/defender-endpoint/post-ti-indicator.md

```diff
@@ -74,9 +74,9 @@ In the request body, supply a JSON object with the following parameters:
 Parameter|Type|Description
 :---|:---|:---
 indicatorValue|String|Identity of the [Indicator](ti-indicator.md) entity. **Required**
-indicatorType|Enum|Type of the indicator. Possible values are: "FileSha1", "FileSha256", "IpAddress", "DomainName" and "Url". **Required**
-action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "AlertAndBlock", and "Allowed". **Required**
-application|String|The application associated with the indicator. **Optional**
+indicatorType|Enum|Type of the indicator. Possible values are: "FileSha1", "FileMd5", "CertificateThumbprint", "FileSha256", "IpAddress", "DomainName" and "Url". **Required**
+action|Enum|The action that will be taken if the indicator will be discovered in the organization. Possible values are: "Alert", "Warn", "Block", "Audit, "BlockAndRemediate", "AlertAndBlock", and "Allowed". **Required**
+application|String|The application associated with the indicator. This field only works for new indicators. It will not updated the value on an existing indicator. **Optional**
```

Comment:
```suggestion
application|String|The application associated with the indicator. This field only works for new indicators. It will not update the value on an existing indicator. **Optional**
```